minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/Client/MalmoModClient.java

```diff
@@ -200,7 +200,7 @@ public void onPressed()
                 // TODO (R): This is a really janky way of extracting constants. This should 
                 // just happen from the command line. -_-
                 // Use this if you want to test some code with a handy key press
-                CraftingHelper.dumpMinecraftObjectRules("../../../herobraine/hero/mc_constants.json");
+                CraftingHelper.dumpRichLidarObs("../../../herobraine/hero/lidar_obs.json");
```

Comment:
should we keep the minecraft object rules?

Reply:
good catch - was hacking